go/vt/vttablet/tabletserver/throttle/throttler_exclude_race_test.go

```diff
@@ -69,6 +69,7 @@ func TestProbesPostDisable(t *testing.T) {
 	})
 
 	t.Run("aggregated", func(t *testing.T) {
+		throttler.Disable()
```

Comment:
Not directly related, but the `TestProbesPostDisable/aggregated` test was consistently failing and causing the codecov workflow to fail -- making it very hard to work on this PR.

Another one that was failing very often was `TestGatewayBufferingWhenPrimarySwitchesServingState`, see: https://github.com/vitessio/vitess/pull/14968

Reply:
This should be removed; the test was racy, please see https://github.com/vitessio/vitess/pull/14971 as the appropriate fix.

Reply:
Thanks! I'll revert it. ❤️